clients/fides-js/src/fides.ts

```diff
@@ -104,42 +108,44 @@ async function init(this: FidesGlobal, config: FidesConfig) {
     ...config,
     options: { ...config.options, ...overrides.optionsOverrides },
   };
-  const cookie = {
+  this.cookie = {
     ...getInitialCookie(config),
     ...overrides.consentPrefsOverrides?.consent,
   };
 
   // Keep a copy of saved consent from the cookie, since we update the "cookie"
   // value during initialization based on overrides, experience, etc.
-  const savedConsent: NoticeConsent = {
-    ...cookie.consent,
+  this.saved_consent = {
+    ...this.cookie.consent,
   };
 
   const initialFides = getInitialFides({
     ...config,
-    cookie,
-    savedConsent,
+    cookie: this.cookie,
+    savedConsent: this.saved_consent,
     updateExperienceFromCookieConsent: updateExperienceFromCookieConsentNotices,
   });
   if (initialFides) {
     Object.assign(this, initialFides);
     updateWindowFides(this);
-    dispatchFidesEvent("FidesInitialized", cookie, config.options.debug);
+    dispatchFidesEvent("FidesInitialized", this.cookie, config.options.debug, {
+      shouldShowExperience: this.shouldShowExperience(),
```

Comment:
something to be aware of is that this `shouldShowExperience` could be different than the `shouldShowExperience` on L 147 of this file. So, if a customer is listening / responding to the 1st call, it could be incorrect.  For example, the banner will surface even though the 1st `shouldShowExperience` is false, in the case where the experience returns a new notice with a key that does not yet exist in the cookie.

Reply:
update here- for TCF, this should remain the same across `FidesInitialized` calls, as we rely on the tcf_version_hash to form this calculation. This shouldn't change across these calls, so I think we're good to go! We just want to be aware in case we have non-TCF customers wanting to use this. We have a follow-up ticket to track a larger init events refactor so that this won't be as misleading

Reply:
Yeah, this could cause some issues.. Having `FidesInitialized` event handlers be called twice, potentially with different data, could easily confuse some code that's not expecting it. My preference would be to use a different event name for the first one and perhaps not even pass `shouldShowExperience` to it.